repository/src/main/java/com/epam/esm/entities/Role.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.esm.entities;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "role")
+public class Role extends Identifiable {
+
+    private String name;
```

Comment:
Let`s use @Column annotation, to increase readability

Reply:
added column annotation